### ISSUE TITLE
Empty artist from Telegram on any track

### ISSUE THREAD

**Issue by futpib** [InputUsername/rescrobbled#85]
Both `qdbus` and `playerctl` show `xesam:artist` just fine.
Same `rescrobbled` works fine with `Firefox` on my system.

```
// rescrobbled --version

rescrobbled v0.6.1
```

```
// journalctl --user -fu rescrobbled

Nov 03 18:29:06 futpib-laptop rescrobbled[779855]: ----
Nov 03 18:29:06 futpib-laptop rescrobbled[779855]: Now playing:  - Plein Soleil ()
Nov 03 18:29:06 futpib-laptop rescrobbled[779855]: Failed to update status on Last.fm
Nov 03 18:29:06 futpib-laptop rescrobbled[779855]: Caused by:
Nov 03 18:29:06 futpib-laptop rescrobbled[779855]:     Now playing request failed: Status code 400 Bad Request indicates failure
```

```
// qdbus org.mpris.MediaPlayer2.tdesktop /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.Metadata

mpris:artUrl: 
mpris:length: 398000000
mpris:trackid: 
xesam:artist: World's End Girlfriend
xesam:title: Plein Soleil
```

```
// playerctl -p tdesktop metadata

tdesktop mpris:artUrl              
tdesktop mpris:length              398000000
tdesktop mpris:trackid             '/org/desktop_app/track/0'
tdesktop xesam:artist              World's End Girlfriend
tdesktop xesam:title               Plein Soleil
```

**Comment by InputUsername** [comment]
Thanks for reporting, this is definitely a bug. I see you've already created a PR to address it in `mpris-rs`, so let's wait until that is merged.

**Comment by InputUsername** [comment]
Just released version 0.6.2 that fixes this!
